active/003-seq-feature.md

```diff
@@ -0,0 +1,126 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage, modification, and querying of features covering a region of a biological sequence. A feature is defined as a sub-region of a seuqence that is a functional entity, e.g., a gene, a riboswitch, an exon, etc.
```

Comment:
Agreed

On Tue, May 3, 2016 at 11:16 PM, Zech Xu <notifications@github.com> wrote:

> In active/003-seq-feature.md
> <https://github.com/biocore/scikit-bio-rfcs/pull/9#discussion_r61993498>:
>
> > @@ -0,0 +1,126 @@
> > +---
> > +Feature name: interval-metadata
> > +Start date: <2015-12-18>
> > +Pull request: 1
> > +Authors: ["@RNAer", "@mortonjt"]
> > +Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
> > +---
> > +
> > +# Summary
> > +
> > +Introduce an `interval_metadata` object to allow for the storage, modification, and querying of features covering a region of a biological sequence. A feature is defined as a sub-region of a seuqence that is a functional entity, e.g., a gene, a riboswitch, an exon, etc.
>
> It mentioned that interval tree data structure could be probably useful in
> cases other than biological sequences in the previous discussion, but not
> the whole set of metadata proposal. do you have other use cases in mind? We
> don't want to over generalize a structure without solid use cases, right?
> Plus, Evan was concerned about feature was loosely connected to interval
> metadata - that's why we further modified the proposal to have tighter
> coupling.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio-rfcs/pull/9/files/3785678436120d960d38cbcaee41fa4e0372f242#r61993498>
>